src/renderer/families/bitcoin/SendAmountFields.js

```diff
@@ -66,57 +70,87 @@ const Fields = ({
     updateTransaction(t => bridge.updateTransaction(t, { rbf: true }));
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const onFeeStrategyClick = useCallback(
+    ({ amount, feesStrategy }) => {
+      onChange(bridge.updateTransaction(transaction, { feePerByte: amount, feesStrategy }));
```

Comment:
we should not use bridge.updateTransaction anymore, instead `updateTransaction` should be used in order to not create race-condition issues.